src/frontends/paddle/src/op/partial_sum.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
since the most part of mapping codes can be reused, can wen consolidate them ? e.g elementwise ops

Reply:
done